### PR TITLE
Add support for Idol of Yogg-Saron

### PR DESCRIPTION
**Pull request by st-htmn**
Hi,

really love your addon, only thing I was kinda missing was support for the Yogg-Saron Idol.
I quickly threw together support for:

- Time on Yogg-Saron Buff
- Number of Stacks
- Number of remaining Stacks until next Thing from Beyond
- Time on Thing from Beyond

Changes were tested ingame and seemed to work fine.

If you are happy with the changes feel free to merge otherwise just close the PR :)

### CODE REVIEW

**Comment by Twintop** [comment]
Hey @st-htmn ! Thanks for taking the time to add this functionality to the bar! I've made some **very** minor tweaks around variable ordering (for when the bar creates the help menu when editing Bar Text) and merged it in.

Hopefully it wasn't too painful to figure out how to get this all added in -- I'm working on cleaning it up to make it way easier than it currently is. Hopefully it'll be smoother next time you contribute.

Thanks again!

File: Functions/News.lua [hunk 1]
```diff
@@ -8,6 +8,23 @@ local oUi = TRB.Data.constants.optionsUi
...
+
+- [#315 - PR - st-htmn](#317) Add support for Idol of Yogg-Saron.
+<br/>&emsp;&ensp;- Thanks go out to *st-htmn* for adding this functionality!
+<br/>&emsp;&ensp;- Tracks the current stacks and time remaining on the Idol of Yogg-Saron buff and the time left on Thing From Beyond once spawned.
+<br/>&emsp;&ensp;- New bar text icon and variable:
+<br/>&emsp;&ensp;&emsp;&ensp;- `#ys` or `#idolOfYoggSaron` -- Idol of Yogg-Saron buff icon
+<br/>&emsp;&ensp;&emsp;&ensp;- `#tfb` or `#thingFromBeyond` -- Thing From Beyond buff icon
+<br/>&emsp;&ensp;&emsp;&ensp;- `$ysTime` -- time remaining on the Idol of Yogg-Saron buff
+<br/>&emsp;&ensp;&emsp;&ensp;- `$ysStacks` -- stacks on the Idol of Yogg-Saron buff
+<br/>&emsp;&ensp;&emsp;&ensp;- `$ysTime` -- stacks required for Idol of Yogg-Saron to spawn a Thing From Beyond
```
Comment (st-htmn) [inline Functions/News.lua:23]: minor typo here, this should be ysRemainingStacks :)

Comment (st-htmn) [reply Functions/News.lua:23]: idk if you want to fix that directly on the dragonflight branch

Comment (Twintop) [reply Functions/News.lua:23]: Whoops. Good catch! Fixed.

**Pull request merged**